### ISSUE TITLE
Problem with NeighMoving copy constructor

### ISSUE THREAD

**Issue by nicolas-clausolles** [gstlearn/gstlearn#88]
I am not sure why, but it seems that the _bipts member is not copied properly.

For example, the following code ends up in a segmentation fault :
```python
import gstlearn as gl

faults = gl.Faults()
faults.addFault(gl.PolyLine2D([0, 0], [0, 1]))
target = gl.BiTargetCheckFaults(faults)
neigh = gl.NeighMoving()
neigh.addBiTargetCheck(target)
print(f"Original target: {type(neigh.getBipts(0))}")  # Ok: Original target: <class 'gstlearn.ABiTargetCheck'>
neigh_copy = gl.NeighMoving(neigh)
print("Copy created")  # Ok
print(f"Copied target: {type(neigh_copy.getBipts(0))}")  # Segmentation fault
```

**Comment by drenard77810** [comment]
Thanks for your help. This bug has been fixed and correction will be available in next publication

**Comment by drenard77810** [comment]
This has been fixed with #89 in branch dev.
